test/conformance/kuttl/generate/clusterpolicy/standard/clone/sync/cpol-clone-list-sync-create/00-cluster-policy.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
This is the file I meant, @prateekpandey14. It's using index `00`.

Reply:
ok, changed to use `01`